src/audio/pcm_converter/pcm_converter_hifi3.c

```diff
@@ -777,7 +781,11 @@ static void pcm_convert_f_to_s24_lin(const void *psrc, void *pdst,
 
 		/* shift and round */
 		x = x * ratio;
+	#if defined(XT_FIROUND_S)
+	    y1 = XT_TRUNC_S(XT_FIROUND_S(x),15)
```

Comment:
Indentation looks wrong here and in the other places. Tabs are used with size 8.

Reply:
Some obvious problems, I will update later